components/PosesList/index.js

```diff
@@ -0,0 +1,69 @@
+import Heading from "../Heading";
+import { getAllPoses } from "@/public/lib/poses";
+import styled from "styled-components";
+
+export const PoseList = styled.li`
+  list-style: none;
+  text-align: center;
+  background-color: #dbb290;
+  border-radius: 15px;
+  padding-bottom: 20px;
+  height: fit-content;
+  box-shadow: 17px 17px 23px -8px rgba(89, 49, 31, 0.52);
+`;
+
+export const PoseImage = styled.img`
+  margin: auto;
+  padding: auto;
+`;
+
+export const Pose = styled.div`
+  // list-style: none;
```

Comment:
The purpose of this element is not very clear, what about
```suggestion
export const PoseContainer = styled.div`
```
?

Reply:
looks good, thank you! 😋